src/rng/test/ut_Engine.cpp

```diff
@@ -179,7 +179,7 @@ template <typename EType> void doit() {
   e5.seed((rtype)99);
   ASSERTEQ(e4, e5);
 
-#if R123_USE_STD_RANDOM
+#ifdef R123_USE_STD_RANDOM
```

Comment:
There are some warnings on ccscs2 for this test.

Reply:
Marked as WIP until I can fix this issue.

Reply:
Adding the headers and implementation headers to Xcode and VS are not strictly required for compilation, they are added as a convenience so the files show up in the 'project explorer' interface.  To simplify the implementation, I combined all headers into a single list.

Reply:
I'm seeing different warnings for different machines.  I've pushed a change that should fix issues on ccscs2, but different warnings may show up on other machines.